packages/cli/src/models/network/NetworkAppController.js

```diff
@@ -148,6 +164,28 @@ export default class NetworkAppController extends NetworkBaseController {
     log.error(`Possible migration method 'migrate' found in contract ${contractClass.contractName}. Remember running the migration after deploying it.`);
   }
 
+  _fetchOwnedProxies(packageName, contractAlias, proxyAddress) {
```

Comment:
is there a need to specify `owned`? is there a chance for a project to manage other proxies than the ones it owns?

Reply:
After changing ownership of a proxy, the project will still keep track of it, and they will still be listed in the network file. It doesn't manage them, but it does have a reference to them.